tests/E2E/test/campaigns/common_scenarios/product.js

```diff
@@ -490,7 +490,7 @@ module.exports = {
     await client.signInFO(AccessPageFO);
     await client.changeLanguage();
     await client.scrollWaitForExistAndClick(productPage.see_all_products);
-    for (let i = 0; i <= global.pagination; i++) {
+    for (let i = 1; i <= global.pagination; i++) {
       for (let j = 0; j < global.productInfo.length; j++) {
         await client.pause(2000);
```

Comment:
Isn't the test gonna be quite long if we wait for 2seconds at each loop? (note that I'm no expert in js tests so maybe it's a dumb question ^^)

Reply:
I can't comment under but juste to be sure since your stated your index at `1` instead of `0`, is the following test still correct? `if (i < pagination)`

Reply:
@jolelievre thank you for your review :) 
Yes i can change it by => (let i = 0; i <global.pagination; i ++) it's the same result.
And about waiting 2 seconds, just after the pause(2000) we have the isVisible function, we need a break each loop to get a good result.

Reply:
@jolelievre Done!